structuredheadings/dialogs/selectstyle.js

```diff
@@ -1,13 +1,106 @@
+var selectedStyle;
+
+var columns = 4;
+var htmlContent = "";
+htmlContent += "<div>";
+htmlContent += "<table role=\"listbox\"";
+htmlContent += "style=\"width:100%;height:100%;border-collapse:separate;\"";
+htmlContent += "cellspacing=\"2\" cellpadding=\"2\"";
+if (CKEDITOR.env.ie && CKEDITOR.env.quirks) {
+  htmlContent += "style=\"position:absolute;\"";
+}
+htmlContent += "><tbody>";
```

Comment:
This looks fairly gross and a pain to edit or find anything in.  :)  I wonder if it would be nicer (although less performant) to just create the actual elements (and do all this table construction using actual elements,) then if CKEditor requires a html string for the dialog, just convert it to a string with getOuterHtml()?  Thoughts?  It seems there would be a fair amount of tedium involved in changing.